src/styles/units/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$watch-size: 400px;
+
+$el-size-sec: 4px;
+$el-size-min: 8px;
+$el-size-dotted: 20px;
+
+$arrows-color: #0d15b7;
+
+$dotted-color: #0d15b7;
+$dotted-border-color: #6552bc;
+
+$style-for-arrows: 50%;
```

Comment:
it is not obvious variable naming. Use meaningful names

Reply:
Changed